makefiles/pacman.mk

```diff
@@ -3,27 +3,44 @@ $(error Use the main Makefile)
 endif
 
 SUBPROJECTS    += pacman
-PACMAN_VERSION := 5.2.2
-DEB_PACMAN_V   ?= $(PACMAN_VERSION)-2
+PACMAN_VERSION := 6.0.0
+DEB_PACMAN_V   ?= $(PACMAN_VERSION)
 
 pacman-setup: setup
 	wget -q -nc -P $(BUILD_SOURCE) https://git.archlinux.org/pacman.git/snapshot/pacman-$(PACMAN_VERSION).tar.gz
```

Comment:
Get the tarball from [here.](https://sources.archlinux.org/other/pacman/) The signature for the tarball is also there, so add that to the Makefile as well.